.github/workflows/cypress.yml

```diff
@@ -6,19 +6,22 @@ on:
   push:
     branches:
       - '**'
+  # Allows workflow to be called from other workflows
+  workflow_call:
+    inputs:
+      ref:
+        required: true
+        type: string
 
 # Avoid duplicate workflows on same branch
 concurrency: 
-  group: ${{ github.workflow }}-${{ github.ref }}
+  group: ${{ github.workflow }}-${{ github.ref }}-cypress
```

Comment:
The Nightly Build workflow calls the Cypress workflow. As a result, `${{ github.workflow }}` in this file is `Nightly Build`. This would cause all the called workflows (cypress, js tests, python tests & py prod smoke tests) to have the same concurrency group. Added the `-cypress` extension to prevent unwanted cancellation of workflows. 

Docs [link](https://docs.github.com/en/actions/using-jobs/using-concurrency#example-only-cancel-in-progress-jobs-or-runs-for-the-current-workflow)